components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -4292,7 +4289,18 @@ public static String getMountedPath(RegistryContext registryContext, String path
         }
         Resource apiDocResource;
         Registry registryType = null;
+        boolean isTenantFlowStarted = false;
         try {
+            int tenantId;
+            if (tenantDomain != null && !MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
+                isTenantFlowStarted = true;
```

Comment:
Fixed in #1494(https://github.com/wso2/carbon-apimgt/commit/26d642b9148c5a2b553209df8060c4857400ee4f)